client/app/stores/message_store.coffee

```diff
@@ -300,6 +301,14 @@ class MessageStore extends Store
                     _messages = _messages.remove message.id
                 else
                     onReceiveRawMessage message
+
+                # Update conversation length
+                conversationID = {message}
+                currentLength = _conversationLengths.get conversationID
+                newLength = currentLength - 1
+                console.log currentLength, newLength
+                _conversationLengths = _conversationLengths
+                                                 .set(conversationID, newLength)
```

Comment:
@jsilvestre you forgot a `console.log`. And, `get + set` is an `update`, isn't it?